lib/trento_web/controllers/v1/cluster_controller.ex

```diff
@@ -16,6 +28,16 @@ defmodule TrentoWeb.V1.ClusterController do
     fallback: TrentoWeb.FallbackController
 
   plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true
+
+  plug TrentoWeb.Plugs.OperationsPolicyPlug,
+       [
+         policy: Trento.Operations.ClusterPolicy,
+         resource: &__MODULE__.get_operation_cluster/1,
+         operation: &__MODULE__.get_operation/1,
+         assigns_to: :cluster
```

Comment:
Unless I am getting it wrong, I am afraid we are missing a `params: &some_func/1` to provide the resource for which we are requesting the maintenance to the cluster operation policy.

Currently we are always falling back to the last `authorize_operation` which will always allow the operation ignoring the current state of the cluster. 
```elixir
def authorize_operation(:cluster_maintenance_change, _, _), do: :ok
```

I believe that is not what we mean to do 😄